tests/components/tomorrowio/test_weather.py

```diff
@@ -99,13 +99,13 @@ async def test_v4_weather(hass: HomeAssistant) -> None:
         ATTR_FORECAST_TEMP: 45.9,
         ATTR_FORECAST_TEMP_LOW: 26.1,
         ATTR_FORECAST_WIND_BEARING: 239.6,
-        ATTR_FORECAST_WIND_SPEED: 9.49,
+        ATTR_FORECAST_WIND_SPEED: 34.16,  # 9.49 m/s -> km/h
     }
     assert weather_state.attributes[ATTR_FRIENDLY_NAME] == "Tomorrow.io - Daily"
     assert weather_state.attributes[ATTR_WEATHER_HUMIDITY] == 23
     assert weather_state.attributes[ATTR_WEATHER_OZONE] == 46.53
-    assert weather_state.attributes[ATTR_WEATHER_PRESSURE] == 3035.0
+    assert weather_state.attributes[ATTR_WEATHER_PRESSURE] == 30.35
```

Comment:
This fixes a bug AFAICT, 30.35 is a reasonable value in inHg, but was previously interpreted as hPa and converted to Pa.